test/test/cpu/CpuTests.java

```diff
@@ -23,7 +24,7 @@ private static void assertCloseTo(long value, long target, String message) {
 
     private static void pinCpu(TestProcess p, int cpu) throws Exception {
         String[] tasksetCmd = {"taskset", "-acp", String.valueOf(cpu), String.valueOf(p.pid())};
-        ProcessBuilder cpuPinPb = new ProcessBuilder(tasksetCmd).inheritIO();
+        ProcessBuilder cpuPinPb = new ProcessBuilder(tasksetCmd).redirectError(Redirect.INHERIT);
```

Comment:
Does `stdout` need to be redirected to `/dev/null`? Otherwise child process may get stuck if output is not consumed.